sample/SparkSummit_17/config.json

```diff
@@ -0,0 +1,5 @@
+{
+  "Email": "spark@gmail.com",
+  "App_Name": "Spark Summit 2017",
+  "Api_Link": "https://raw.githubusercontent.com/arp95/open-event/spark/sample/spark"
```

Comment:
The API link isn't correct. Please fix

Should be https://raw.githubusercontent.com/arp95/open-event/spark/sample/SparkSummit_17